grpc-api/src/main/proto/com/example/grpcapi/service/device_service.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package com.example.grpcapi.service;
+
+import "com/example/grpcapi/reqrep/device/get_by_id.proto";
+import "com/example/grpcapi/reqrep/device/get_updated_devices.proto";
+import "com/example/grpcapi/reqrep/device/create.proto";
+import "com/example/internal/input/reqreply/device/update.proto";
+
+service DeviceService {
+  rpc createDevice(grpcapi.reqrep.device.CreateDeviceRequest) returns
+      (grpcapi.reqrep.device.CreateDeviceResponse);
+
+  rpc getDeviceById(grpcapi.reqrep.device.GetDeviceByIdRequest) returns
+      (reqrep.device.GetDeviceByIdResponse);
+
+  rpc subscribeToUpdateByUserId (grpcapi.reqrep.device.GetUpdatedDevicesRequest)
+      returns (stream internal.input.reqreply.device.UpdateDeviceResponse);
```

Comment:
Better change response from internal.input.reqreply.device.UpdateDeviceResponse to commonmodels.device.Device.